dotcom-rendering/src/server/handler.article.amp.tsx

```diff
@@ -1,6 +1,6 @@
 import { ArticleDesign } from '@guardian/libs';
 import type { RequestHandler } from 'express';
-import { Standard as ExampleArticle } from '../../fixtures/generated/articles/Standard';
+import { Standard as ExampleArticle } from '../../fixtures/generated/dcr-articles/Standard';
```

Comment:
If this is used to mock a frontend request for the performance test handler should it be using `fe-articles`?

```suggestion
import { Standard as ExampleArticle } from '../../fixtures/generated/fe-articles/Standard';
```

Reply:
Good spot thanks. Have updated all the affected server handlers